lib/custom_cops/duffy.rb

```diff
@@ -0,0 +1,10 @@
+# typed: false
+# frozen_string_literal: true
+
+module CustomCops
```

Comment:
I don't think this file should be in the `lib` folder, since it is not relevant to the operation of the gem. IMO, it shouldn't even be packaged into the gem bundle.